.gitignore

```diff
@@ -35,3 +35,4 @@ __pycache__
 
 # logs
 *.log
+id_rsa
```

Comment:
Is there a reason this was added outside of personal configuration? I'm not sure that your rsa_id should be located within this repository's file structure

Reply:
if I placed it anywhere else I couldnt ssh tunnel.